app/controllers/feedbacks_controller.rb

```diff
@@ -23,7 +24,8 @@ def feedback_params
   end
 
   def retrieve_feedback
-    safe_params = params.permit(:id)
-    Feedback.find(safe_params[:id])
+    safe_params = params.permit(:id, :klass)
+    klass = safe_params[:klass].constantize
+    klass.find(safe_params[:id])
```

Comment:
Huhu, c’est work in progress ça, non?

👍 pour réutiliser le même controller, par contre passer la `klass` en paramètres, ça me parait un peu limite niveau sécurité. Idée: est-ce qu’on pourrait gérer les deux ressources avec le même controller, et choisir le modèle en fonction du path?